recipes/litellm_ad/meta.yaml

```
@@ -0,0 +1,89 @@
+{% set name = "litellm_ad" %}
+{% set version = "1.76.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/litellm_ad-{{ version }}.tar.gz
+  sha256: 18f6bd097a35af5c58d8b08430adf05f2e659a3b1b9dfa7d422b7b5be2d579ff
+
+build:
+  entry_points:
+    - litellm = litellm:run_server
+    - litellm-proxy = litellm.proxy.client.cli:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - poetry-core
+    - wheel
+    - pip
+  run:
+    - python >=3.8.1,<4.0,!=3.9.7
```

```suggestion
    - python >={{ python_min }},<4.0
```

current default `python_min` is 3.10, so simplifying this.